.github/workflows/go.yml

```diff
@@ -26,3 +26,9 @@ jobs:
 
     - name: Test
       run: go test -v ./...
+
+    - name: Vet
+      run: go vet -v ./...
+
+    - name: Format
+      run: '[ ! "$(gofmt -s -l .)" ]'
```

Comment:
let's also use goimports which i think is just called with `-l .`

Reply:
Added.